packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -174,7 +174,9 @@ describe("Runtime", () => {
 				return opFakeSequenceNumber++;
 			},
 			submitSignalFn: (content: unknown, targetClientId?: string) => {
-				submittedSignals.push(content as ISignalEnvelope); // Note: this object shape is for testing only. Not representative of real signals.
+				if (targetClientId === undefined || targetClientId === mockClientId) {
+					submittedSignals.push(content as ISignalEnvelope); // Note: this object shape is for testing only. Not representative of real signals.
+				}
```

Comment:
I removed filtering here and opted for your idea [here](https://github.com/microsoft/FluidFramework/pull/22321#discussion_r1748845091): 'Maybe we want to have the model where process* means service looks are what it has and sends signals down and things are routed to runtime.' 

Added a test that simulates no service support for targeted signals (i.e. marked as targeted/expected not to return on the client side but still comes back).

Reply:
note: I also remember a comment on the last PR saying the Notifications feature of DIS would benefit from being able to see that a targeted signal sent without service support came back. So, I think broadcasting rather than limiting will be the plan with no service support.